ibis/expr/types/json.py

```diff
@@ -1,34 +1,36 @@
 from public import public
 
-from ibis.expr.types.binary import BinaryColumn, BinaryScalar, BinaryValue
-from ibis.expr.types.strings import StringColumn, StringScalar, StringValue
+from ibis.expr.types import Column, Scalar, Value
 
 
 @public
-class JSONValue(StringValue):
-    pass  # noqa: E701,E302
+class JSONValue(Value):
+    def __getitem__(self, key):
```

Comment:
Could you please add a follow-up to provide `.get(key, default)` API similar to maps?